ZenPacks/zenoss/ZenPackLib/lib/zuul.py

```diff
@@ -59,12 +59,13 @@
 from Products.Zuul.facades.processfacade import ProcessFacade
 from Products.Zuul.facades.servicefacade import ServiceFacade
 
-from Products.Zuul.catalog.paths import (
-     DevicePathReporter,
+from Products.Zuul.catalog.paths import DevicePathReporter
```

Comment:
Actually these changes are needed since the imports are coming from the ZPL overrides for each of the path reporters